crates/rng/src/main.rs

```diff
@@ -161,10 +162,13 @@ pub(crate) fn start_backend(config: VuRngConfig) -> Result<()> {
     Ok(())
 }
 
-fn main() -> Result<()> {
+fn main() {
     env_logger::init();
 
-    start_backend(VuRngConfig::try_from(RngArgs::parse()).unwrap())
+    if let Err(e) = start_backend(VuRngConfig::try_from(RngArgs::parse()).unwrap()) {
```

Comment:
```suggestion
    if let Err(e) = VuRngConfig::try_from(RngArgs::parse()).and_then(start_backend) {
```

Not important, just a suggestion

Reply:
Will add this to the followup PR. Thanks!

Reply:
Changed my mind and moved some commits (along with this suggestion) over from the followup PR in order to simplify the dependencies between the changes.